common/djangoapps/third_party_auth/models.py

```diff
@@ -204,6 +212,13 @@ def get_authentication_backend(self):
         """Gets associated Django settings.AUTHENTICATION_BACKEND string."""
         return '{}.{}'.format(self.backend_class.__module__, self.backend_class.__name__)
 
+    def available_for_login(self):
```

Comment:
Also, this could be a `property` for more consistency with self.enabled and self.accepts_logins and self.visible